frontend/src/components/utils/HeaderWithCloseButton.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+    import { createEventDispatcher } from 'svelte';
+    const dispatch = createEventDispatcher();
```

Comment:
dispatch is missing a type

Reply:
createEventDispatcher is a function imported from svelte library. Don't really know what time to put. Function?